<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="WidgetOpenGL" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>521</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="manhatan">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Manhatan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="euclidiana">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Euclidiana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pearson">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pearson</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coseno">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Coseno</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minkowsky">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Minkowsky</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jaccard">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Jaccard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>knn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valorK">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario 1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario 2 :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="user2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="valorMink">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dataBase">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MovieRatings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MovieLensSmall</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Libros</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jester</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base de Datos</string>
    </property>
   </widget>
   <widget class="QComboBox" name="distancia">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Manhatan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euclidiana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Minkowsky</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pearson</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jaccard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coseno</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listResultados">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cosenoAjustado">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Coseno Ajustado</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tipoCosenoAjustado">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>prod</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>user</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Matriz">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>551</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetOpenGL</class>
   <extends>QWidget</extends>
   <header location="global">widgetopengl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
